packages/kv-store/composer.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "tempest/kv-store",
```

Comment:
I'd call it `key-value-store`

Reply:
"KV" is the standard term for those (eg. [Cloudflare's](https://developers.cloudflare.com/kv/)), but if you want, we can change it. Just confirm it to me

Reply:
oh ok. It's not a big issue, I'm fine with `kv` as well